e2e/address_types.test.js

```diff
@@ -12,12 +12,12 @@ describe('Example', () => {
   it('loads my test wallet', async () => {
     // let newWalletButton  element(by.text('CREATE NEW WALLET (NEW SEED)'));
     await element(by.id("loadingapp.restorewalletseed")).tap();
-    await element(by.id("seed.seedplaceholder")).typeText('lottery multiply patient simple ivory leisure swift square west despair beauty match crowd margin reject box always title photo remind word diet ecology badge');
-    await element(by.id("birthdayinput")).typeText('1985000');
+    await element(by.id("seed.seedplaceholder")).replaceText('lottery multiply patient simple ivory leisure swift square west despair beauty match crowd margin reject box always title photo remind word diet ecology badge');
```

Comment:
Where'd this test seed come from?

Reply:
This brings up a bigger issue, which that we don't have a plan for referencing a Single Source of Truth for our test data.

Here's a place where one can go to find a seed that we use in the integration tests:

https://github.com/zingolabs/zingolib/blob/d23559d22f341002440c0dc9acffc292bf110438/zingocli/tests/data/mod.rs#L11

But probably the correct thing for us to do is to have a repository of data that each of our different test frameworks can reference.   A nice thing about doing it this way is that we can then share test "vectors" across projects.

This is an important kind of DRYness.
I think it might be best to look at how these folks do it:

https://github.com/ZcashFoundation/

Reply:
Since `zingo-moble` already depends on `zingolib` I think the right answer is to use those test vectors I linked.

Reply:
Test data is non-blocking.